src/main/java/com/nerodesk/App.java

```diff
@@ -100,113 +102,121 @@ public App(final Base base) throws IOException {
      */
     @SuppressWarnings("PMD.DoNotUseThreads")
     public static Takes make(final Base base) throws IOException {
-        final Takes fork = new TsFork(
-            new FkParams(
-                PsByFlag.class.getSimpleName(),
-                Pattern.compile(".+"),
-                new TkRedirect()
-            ),
-            new FkRegex(
-                "/xsl/.*",
-                new TsWithType(new TsClasspath(), "text/xsl")
-            ),
-            new FkRegex(
-                "/css/.*",
-                new TsWithType(
+        final Takes fork = new TsFallback(
```

Comment:
@longtimeago method `make()` seems to be too huge and completely unreadable, could you please re-design it? E.g. extracting creation of `FkRegex`'s to separate methods?